string obj
hirop_msgs/Pose pose
---
int8 result

